src/local_scheduler/format/local_scheduler.fbs

```diff
@@ -36,6 +36,11 @@ enum MessageType:int {
   PutObject
 }
 
+// This message is sent from a worker to a local scheduler.
+table GetTaskRequest {
+  task_success: bool;
```

Comment:
What does it mean if `task_success` is True versus False? And why is this necessary?

Reply:
This is a little messy, but it's used to notify the local scheduler of the actor's new task counter. For checkpoint tasks, the actor's task counter should only be updated if the task was successful (the checkpoint was actually saved or resumed). If the checkpoint isn't there, for example, then `task_success` is set to `False` and the local scheduler won't modify the actor's task counter.